test/libsolidity/util/TestFileParserTests.cpp

```diff
@@ -305,6 +305,24 @@ BOOST_AUTO_TEST_CASE(call_arguments_tuple)
 	testFunctionCall(calls.at(1), Mode::SingleLine, "f((uint8),uint8)", false);
 }
 
+BOOST_AUTO_TEST_CASE(call_arguments_left_aligned)
+{
+	char const* source = R"(
+		// f(bytes32, bytes32): 0x6161, 0x420000EF -> 1
```

Comment:
Please add a test that shows that it works with explicit leading or trailing zeros.

What about an odd number of hex digits?

Reply:
> Please add a test that shows that it works with explicit leading or trailing zeros.

Done.
> What about an odd number of hex digits?

Does not work. Need to fix that ;)

Reply:
It should not be valid!